packages/kmath/README.md

```diff
@@ -370,38 +399,46 @@ few equalityTolerances could appear in the wrong order.
 Returns a polar point `[length, angle]` from the two-dimensional cartesian
 point `v`, where `angle` is measured in radians.
 
-    kmath.point.polarRadFromCart([1, 1])
-    => [1.4142135623730951, 0.7853981633974483]
+```js
+kmath.point.polarRadFromCart([1, 1])
+=> [1.4142135623730951, 0.7853981633974483]
+```
 
 #### `point.polarDegFromCart(p)`
 
 Returns a polar point `[length, angle]` from the two-dimensional cartesian
 point `v`, where `angle` is measured in degrees.
 
-    kmath.point.polarDegFromCart([0, 2])
-    => [2, 90]
+```js
+kmath.point.polarDegFromCart([0, 2])
+=> [2, 90]
+```
 
 #### `point.cartFromPolarRad(polar)` or `point.cartFromPolarRad(length, angle)`
 
 Returns a two-dimensional cartesian point from the polar point input,
 where the input angle is measured in radians.
 
-    kmath.point.cartFromPolarRad([2, Math.PI])
-    => [-2, 0]  // Note: a very small nonzero number is actually returned here,
-                // due to numerical inaccuracy.
-    kmath.point.cartFromPolarRad(Math.pow(2, 0.5), Math.PI/4)
-    => [1, 1]
+```js
+kmath.point.cartFromPolarRad([2, Math.PI])
+=> [-2, 0]  // Note: a very small nonzero number is actually returned here,
+            // due to numerical inaccuracy.
+kmath.point.cartFromPolarRad(Math.pow(2, 0.5), Math.PI/4)
+=> [1, 1]
+```
 
 #### `point.cartFromPolarDeg(polar)` or `point.cartFromPolarDeg(length, angle)`
 
 Returns a two-dimensional cartesian point from the polar point input,
 where the input angle is measured in degrees.
 
-    kmath.point.cartFromPolarRad([2, 90])
-    => [-2, 0]  // Note: a very small nonzero number is actually returned here,
-                // due to numerical inaccuracy.
-    kmath.point.cartFromPolarRad(Math.pow(2, 0.5), 45)
-    => [1, 1]
+```js
+kmath.point.cartFromPolarRad([2, 90])
```

Comment:
It looks like this form isn't supported anymore.  Can you remove this from the docs?